Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -68,24 +46,59 @@ namespace O3DE::ProjectManager
         footerLayout->addWidget(m_backNextButtons);
         vLayout->addWidget(footerFrame);
 
-
         m_backButton = m_backNextButtons->addButton(tr("Back"), QDialogButtonBox::RejectRole);
         m_backButton->setProperty("secondary", true);
         m_nextButton = m_backNextButtons->addButton(tr("Next"), QDialogButtonBox::ApplyRole);
 
-        connect(m_tabWidget, &TabWidget::currentChanged, this, &CreateAGemScreen::ConvertNextButtonToCreate);
+        connect(m_tabWidget, &TabWidget::currentChanged, this, &CreateAGemScreen::UpdateNextButtonToCreate);
         connect(m_backButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleBackButton);
         connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleNextButton);
 
-        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnGemDisplayNameUpdated);
-        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnGemSystemNameUpdated);
-        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnLicenseNameUpdated);
-        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnCreatorNameUpdated);
-        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnRepositoryURLUpdated);
-
         setObjectName("createAGemBody");
         setLayout(vLayout);
+    }
 
+    void CreateAGemScreen::LoadButtonsFromGemTemplatePaths(QVBoxLayout* firstScreen)
+    {
+        QHash<QString, QString> formattedNames;
+        formattedNames.insert(QString("Prebuilt Gem Tenokate"), QString(tr("Prebuilt Gem")));
```

Comment:
Yep the only reason is that the names are incorrect. I did not want to touch the data files because I didn't know if it would break anything down the line.